include/upgrader/streams/core/e7038ce9-8cd26958.task.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * Add all indexes we've implemented over time. Check if exists
+ * to avoid errors.
+ */
+class AddIndexMigration extends MigrationTask {
+    var $description = "Add indexes accumulated over time";
+    var $indexes = array(
+        'department:flags',
+        'draft:staff_id',
+        'draft:namespace',
+        'file:type',
+        'file:created',
+        'file:size',
+        'form:type',
+        'form_field:form_id',
+        'form_field:sort',
+        'queue:staff_id',
+        'queue:parent_id',
+        'staff:isactive',
+        'staff:onvacation',
+        'task:flags',
+        'team_member:staff_id',
```

Comment:
@JediKev I updated my PR to remove the Alter Table statement to add an index. So I'm thinking now you'd need rebase and add ticket:ticket_pid here, right?

Reply:
@aydreeihn Alrighty rooster. Will do.